packages/lmdb/lmdb.1.0/url

```diff
@@ -0,0 +1,5 @@
+src: "https://github.com/Drup/ocaml-lmdb/archive/1.0.tar.gz"
```

Comment:
`url` files are deprecated by opam 2.0. Could you remove it and insert it in a `url` field instead? (see any other packages for the syntax)